src/api/app/models/notification.rb

```diff
@@ -16,6 +16,21 @@ class Notification < ApplicationRecord
 
   scope :stale, -> { where('created_at < ?', 3.months.ago) }
 
+  def self.projects_with_count
```

Comment:
`_with_count`? What are you trying to express?

Reply:
`self.projects_with_count` will return a hash, where the keys are projects, and the values are the number of ocurrences each project happens. So `_with_count` refers to the number or ocurrences of each project.

Reply:
So never mind what data structure this returns, what is this about? Notifications grouped by project?

Reply:
Yes, kind of.

Reply:
So, instead of `projects_with_count`, would it be better `projects_with_number_of_notifications`?

Reply:
How about `projects_with_notifications`?

Reply:
And do the rest of the hash magic in a view helper?

Reply:
'projects_with_count' was renamed to `projects`, and the hash magic code was extracted to a view helper. Please review it again.